TraitsExpanded/TraitSystemBehavior.cs

```diff
@@ -120,26 +159,32 @@ private void GenerateCharacterTraits(CharacterObject character)
 
         public List<ITrait> GetAllPositiveTraits()
```

Comment:
why are the `GetAll . . .()` methods needed?

Reply:
I added them so that when the Player creates a new game and accesses the "trait purchasing" period, Eskalior can easily group them so that you have a list of positive and a list of negative traits